psutil/arch/windows/wmi.c

```diff
@@ -31,6 +31,15 @@ double load_avg_1m = 0;
 double load_avg_5m = 0;
 double load_avg_15m = 0;
 
+#ifdef Py_GIL_DISABLED
+static PyMutex mutex;
+#define MUTEX_LOCK(m) PyMutex_Lock(m)
+#define MUTEX_UNLOCK(m) PyMutex_Unlock(m)
+#else
+#define MUTEX_LOCK(m)
+#define MUTEX_UNLOCK(m)
+#endif
```

Comment:
OK, this is probably a matter of personal taste (and I know it's probably incorrect), but in this case I find it more clear for this to be indented:

```c
#ifdef Py_GIL_DISABLED
    static PyMutex mutex;
    #define MUTEX_LOCK(m) PyMutex_Lock(m)
    #define MUTEX_UNLOCK(m) PyMutex_Unlock(m)
#else
    #define MUTEX_LOCK(m)
    #define MUTEX_UNLOCK(m)
#endif

Reply:
Done!